src/components/atoms/calendarIssueHeader/calendarIssueHeader.jsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react'
 
-const issueHeaderAgenda = ({ cssClass, todayEventsNumber = 0 }) => {
-  const eventText = todayEventsNumber === 0 ? `pas d'évènement prévu` : todayEventsNumber > 1 ? `évènements prévus` : `évènement prévu`
+const issueHeaderAgenda = ({ cssClass, todayEventsNumber = 0, numberInterventions = 0 }) => {
```

Comment:
```suggestion
const issueHeaderAgenda = ({ cssClass, todayEventsNumber = 0, interventionsNumber = 0 }) => {
```
peut etre plus cohérent comme ca ou `numberOfInterventions`

Reply:
j'hésite toujours pour l'ordre haha. Je vais harmoniser ça